README.md

```diff
@@ -61,6 +179,23 @@ Binding preference is determined by the relative orientation of the first and la
 | &rarr; | &darr; | G |
 | &rarr; | &larr; | T |
 
+
+### Authors' Note: 
+A number of ambiguities present in the original program are given defined behavior in this rendition of the system, listed below. 
+
+**Binding Preference of Single Amino Acid Enzymes**
+
+Since the binding preference of an enzyme is determined by the orientation of its first and last element, enzymes which consist of only a single instruction don't have a clearly defined 
+rule for determining their starting base binding preference. We have elected to follow the fairly intuitive convention of having this class of enzymes start on 'A', counting their only element
+as both the first and last fold. With both directions facing right, it follows that the enzyme would bind to base 'A' to begin with. 
+
+**Starting Base Position Preference of Enzymes**
+
+Hofstadter also fails to define which element the enzyme prefers to bind to when given a selection of options. Examples in the book show the first matching element most commonly selected, but 
+this is not an explicitly defined rule. As such, the user is given a selection of bases to choose from when there are multiple matching bases in the strand. The user may also pass 2 parameters to
+the program, --Select-random (-s) and --first-matching (-f) which will have enzymes either randomly choose a starting base to begin on each time, or the first matching element of their starting base 
```

Comment:
Lowercase `--select-random`?

Reply:
Also maybe create a new flag or change the `--first-matching` flag to take a number of which starting option to take. Like, always select the n'th option. `--bind-position n`. First matching would then be `--bind-position 1`

Reply:
> Lowercase `--select-random`?

I need to change the select base flags, they're currently -Sf or -Sr for 'Select first' or 'Select random'. Also there is no long form implementing yet, so entering --Select-random would just throw errors. 



> Also maybe create a new flag or change the `--first-matching` flag to take a number of which starting option to take. Like, always select the n'th option. `--bind-position n`. First matching would then be `--bind-position 1`

I was thinking about this, but it would be more akin to semi randomness. Because each time there is a choice for enzyme binding, the position numbers 1854000changes. So if the enzyme binds to 'A', that might correspond to Positions 1, 5, 10 in the strand. But the next enzyme that binds to 'C' won't have those options, instead they'll have 3,4,6 or whatever. So --bind-position would select array elements instead, and then there really isn't a choice there since it hard to predict what options will be available.